Products/ZPyODBCDA/pyodbc_DA.py

```diff
@@ -106,25 +105,10 @@ def manage_edit(self, title, connection_string, auto_commit=None,
         conn_param['connx_string'] = self.connx_string
         conn_param['auto_commit'] = self.auto_commit
         conn_param['MaxRows'] = self.MaxRows
-        Connection.manage_edit(self, title, repr(conn_param), connected, REQUEST)
+        Connection.manage_edit(self, title, repr(
+            conn_param), connected, REQUEST)
 
     manage_testForm = DTMLFile('dtml/pyodbcconnectionTestForm', globals())
 
     def factory(self):
         return DB
-
-    def sql_quote__(self, v, escapes={
```

Comment:
Why has this been removed? I just read the discussion regarding SQL quoting in `DocumentTemplate` and there was something mentioned along the lines that `DocumentTemplate` should only provide the basic standard SQL quoting rules why specific adapters might implement `sql_quote__` to include dialect specific quoting.

Reply:
I cannot remember removing this. Looks like an accident. I'll put it back in pronto.